keps/sig-scheduling/3902-decoupled-taint-manager/README.md

```diff
@@ -173,7 +173,7 @@ Since there are no new API changes, we can skip Alpha and go to Beta directly.
 
 #### GA
 
-* Fix all reported bugs if any.
+* No negative feedback.
```

Comment:
In the integration and E2E section, please link to the place where the tests are defined, and to the error reports in https://storage.googleapis.com/k8s-triage/index.html

Reply:
@alculquicondor added.